[Java] Dijkstra's Shortest Path Algorithm (Adjacency Matrix)

sumitaccess007 opened the pull request:
Implementation of Dijkastra's Shortest Path Algorithm in Java (Data Structure used is Adjacency Matrix)

Graph
- [x] Dijkstra's algorithm
- [ ] Edmonds–Karp algorithm
- [ ] Ford–Fulkerson algorithm
- [ ] Floyd–Warshall algorithm
- [ ] Tarjan's strongly connected components algorithm

amanmehara commented on Java/Dijkastra Shortest Path Algorithm (Graph in form of Adjacency Matrix)/DijkastraShortestPathAlgorithm.java:
```diff
@@ -0,0 +1,241 @@
+/* Copyright 2018 Aman Mehara
```
Please write your own name and change the year to 2020.

sumitaccess007 replied:
Thanks, This correction has now been taken care.

amanmehara reviewed (changes requested):
Please remove the `Fibonacci Search` source file, as it is aleady merged from different PR.

Also fix the spelling of **Dijkstra**. 😛

sumitaccess007 commented:
> Please remove the `Fibonacci Search` source file, as it is aleady merged from different PR.
> 
> Also fix the spelling of **Dijkstra**. 😛

Hi @amanmehara 
Thanks for the Review, I have removed Fibonacci Search and Corrected Dijkstra Spelling as well.

Thanks to review again.
